examples/transformers_text.py

```diff
@@ -114,6 +162,9 @@ def __call__(self, sentences: List[str]) -> Tensor:
                                  mask).squeeze(1).cpu())
         elif self.pooling == "cls":
             return out.last_hidden_state[:, 0, :].detach().cpu()
+        elif self.pooling == "last":
```

Comment:
in the line `mask = inputs["attention_mask"]`, can you add a shape annotation and describe what it stores?

Reply:
Sure